src/librustc_allocator/expand.rs

```diff
@@ -88,15 +88,15 @@ impl<'a> Folder for ExpandAllocatorDirectives<'a> {
             span,
             kind: AllocatorKind::Global,
             global: item.ident,
-            alloc: Ident::from_str("alloc"),
+            core: Ident::from_str("alloc"),
```

Comment:
s/alloc/core?

Reply:
Yes, thanks. (I thought I’d already fixed that, I probably messed up a rebase.)